glommio/src/executor/mod.rs

```diff
@@ -2314,7 +2516,7 @@ mod test {
         );
         let start = getrusage_utime();
         ex.run(async {
-            Local::local_into(
+            crate::local_into(
```

Comment:
why is this different than the spawn terminology?

Reply:
See my reply above, I had good intentions but I made this more complicated than it needs to be. Will rename to `spawn_*` here as well.

Reply:
This is fixed now :bow: